helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java

```diff
@@ -67,10 +67,17 @@
 
 
 public class MaintenanceManagementService {
+  private static class MaintenanceExecutorService {
+    static final int totalInstance = 1000;
+    static final int faultZones = 20;
```

Comment:
It may be overcomplicating things to have a notion of total instances and fault zones here. If a user is configuring this, they know how many threads they can afford to have in the pool. They may not know how the number of totalInstances and faultZones determine the number of threads without reading the code.